CONTRIBUTING.md

```diff
@@ -1,4 +1,10 @@
 # Contributing
 The bulk of the development process is contained within our Discord group. If you'd like to contribute, consider [joining](https://discordapp.com/invite/NACP6WZ) our group.
 
-Given that this project was conceived with beginners in mind, a section for setting up the programming environment is necessary. For Java development, there are some requirements before diving into it.
\ No newline at end of file
+It is recommended that you first read [github's contribution guide](https://github.com/firstcontributions/first-contributions).
+
+## Issues
+Before starting working on an issue, please comment your intent do so. This is to avoid two different people working on the same issue.
```

Comment:
I think it's possible to assign yourself to an issue, this may be the best way because people can then filter on unassigned issues.

Reply:
I see, yeah, that would delete this problem all together